src/main/java/uk/gov/hmcts/divorce/systemupdate/service/CcdSearchService.java

```diff
@@ -62,15 +65,44 @@ public SearchResult searchForCaseWithStateOf(final State state,
             sourceBuilder.toString());
     }
 
-    public List<CaseDetails> searchForAllCasesWithStateOf(final State state, User user, String serviceAuth) {
+    public SearchResult searchForCasesWithStateOfDueDateBeforeWithoutFlagSet(final State state,
+                                                                             final int from,
+                                                                             final int size,
+                                                                             final String flag,
```

Comment:
would be nice to call this as searchQueryParam or something similar

Reply:
Sorry what do you mean?

Reply:
I mean instead of flag we could call it searchQueryParam :)